capplets/default-applications/mate-da-capplet.c

```diff
@@ -330,7 +331,7 @@ static struct {
 	{"mobility_image",     "preferences-desktop-accessibility"},
 	{"messenger_image",    "user-idle"},
 	{"filemanager_image",  "file-manager"},
-	{"imageviewer_image",  "image-x-generic"},
+	{"imageviewer_image",  "eom"},
```

Comment:
Was it needed to change this icon name? Most icon themes out there don't have a `eom` icon...

Reply:
Hi,
Because Eye Of Mate is part of Mate Desktop, nothing else.

Regards

Reply:
The icon-name should be reverted to "image-x-generic", like "video-x-generic" and "audio-x-generic". Those icons are the generic mime types which can be opened using its default application, the user sets that in the tab below.

![Captura de pantalla a 2019-06-07 23-54-41](https://user-images.githubusercontent.com/10171411/59135628-fd989d00-897f-11e9-895e-78fd251c2215.png)

eom isn't displayed in the list of image viewers like eog does. It's an eom issue.

Reply:
Hi,

Atril is the default office file viewer and purposed like this because it's part of the Mate Desktop, his icon is used in the GUI. In previous version of the PR, i purpose to use another generic icon (not the correct icon sure) and keep the "image-x-generic" too.

I think EOM should be presented like Atril so i purpose to use his icon too for the GUI.

Regards

Reply:
See https://github.com/mate-desktop/mate-control-center/pull/451

Reply:
Ok, seen.